server/models/Request.model.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user: { type: mongoose.ObjectId, required: true, ref: "user" },
+    sitter: { type: mongoose.ObjectId, required: true, ref: "user" },
+    dog: { type: mongoose.ObjectId, required: true, ref: "dog" },
+    start: { type: Date, required: true, trim: true },
+    end: { type: Date, required: true, trim: true },
```

Comment:
If you're interested, you can look into adding validation here to check `end > start` as a bonus 😎